server/container_create.go

```diff
@@ -275,6 +277,19 @@ func setupContainerUser(specgen *generate.Generator, rootfs string, sc *pb.Linux
 		if err1 != nil {
 			return err1
 		}
+		passwdPath, err := generatePasswd(uid, gid, rootfs, runDir)
+		if err != nil {
+			return err
+		}
+		if passwdPath != "" {
+			mnt := rspec.Mount{
+				Type:        "bind",
+				Source:      passwdPath,
+				Destination: "/etc/passwd",
+				Options:     []string{"ro", "bind", "nodev", "nosuid", "noexec"},
+			}
+			specgen.AddMount(mnt)
+		}
```

Comment:
Is it possible to get an empty passwdPath without hitting an error?  If so, probably ought to handle that.

Reply:
Or perhaps add a debug statement for now at least.

Reply:
Empty passwdpath, tells us that the user was in the passwd file, so we do nothing.  We only add the passwd mount point if the user that the container is running with, does not exist in the /etc/passwd of the container.